vector/src/main/res/layout/fragment_ftue_auth_use_case.xml

```diff
@@ -79,7 +79,7 @@
 
         <TextView
             android:id="@+id/useCaseOptionOne"
-            style="@style/Widget.Vector.TextView.Subtitle"
+            style="@style/Widget.Vector.TextView.Subtitle.Medium"
```

Comment:
Note that #3907 is a thing and I have still not found a good solution for it. So basically this change has no effect (I think), which does not mean that we should not do it.

Reply:
I see... the figma typography is a little misleading here then :sweat_smile: Seems like semi bold is the correct weighting